src/App.css

```diff
@@ -0,0 +1,192 @@
+/* stylelint-disable declaration-colon-newline-after */
+
+.NavBar {
+  background-color: #fff;
+  color: black;
+  font-size: 20px;
+  font-weight: bold;
+  text-align: center;
+}
+
+.coins-container {
+  margin: 0;
+  display: flex;
```

Comment:
- [ OPTIONAL ] You could do more following the design on  font face, size, and weight composition and space between elements.

Reply:
ok